tests/flow/constraint_utils.py

```
@@ -0,0 +1,152 @@
+import time
+from index_utils import *
```

Ruff (0.12.2)

`from index_utils import *` used; unable to detect undefined names

(F403)